debug_module.tex

```diff
@@ -222,6 +222,13 @@ \section{Abstract Commands} \label{abstractcommands}
     \end{tabulary}
 \end{table}
 
+Before starting an abstract command, a debugger must ensure that \Fhaltreq,
+\Fresumereq, and \Fackhavereset are all 0.
```

Comment:
I thought \Fackhavereset was W1... how do you ensure that it is 0?

Reply:
ackhavereset is W. The bit has no effect while it is 0.

Reply:
right, so if I write a 1 to DMCONTROL.ackhavereset, and I want to then start an abstract command, must I write DMCONTROL again with the value 0?

Is this a legal sequence:
DMCONTROL => ... ackavereset = 1
ABSTRACTCS => new command

or must I write twice to DMCONTROL?

Reply:
You must write dmcontrol twice in that case.